client/src/components/admin/ResultAttemptsForm/ResultAttemptsForm.jsx

```diff
@@ -96,7 +96,7 @@ function ResultAttemptsForm({
   }
 
   function confirmSubmission() {
-    const submissionWarning = attemptResultsWarning(
+    const [submissionWarning, confirmationKeyword] = attemptResultsWarning(
```

Comment:
Given that the second element is optional, let's make it return an object, so it's more structured. It would be either `{ text: "...", confirmationKeyword: "..." }` or `{ text: "..." }`.

```suggestion
    const submissionWarning = attemptResultsWarning(
```